cmd/api/config.go

```diff
@@ -44,6 +44,15 @@ type config struct {
 	Analytics struct {
 		FetchExtraBlockInfo bool `default:"false"`
 	}
+	Backup struct {
```

Comment:
lmk if we should have all these configurable `Enable` flags or we can simply assume that the backup process will run with all operations for all validator